src/test/util_sds_check/Makefile

```
@@ -0,0 +1,21 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright 2025, Hewlett Packard Enterprise Development LP
+
+#
+# src/test/util_sds_check/Makefile -- shutdown_state_check unit test
+#
+TOP = ../../..
+vpath %.c $(TOP)/src/common/
+vpath %.c $(TOP)/src/test/unittest/
+
+TARGET = util_sds_check
+OBJS = util_sds_check.o\
+	shutdown_state.o
+
+LIBPMEM2=y
+
+include ../Makefile.inc
+
+CFLAGS += -Wall -Wextra -Wno-unused-parameter -Wno-ignored-qualifiers
```

My bad. It is required. But for `ASSERT*()` to work. Sorry.
And I believe non of these here are necessary, are they?

```suggestion
CFLAGS += -DDEBUG
```

---

ASSERT*() work but test doesn't build up
-Wno-unused-parameter is needed to mocked function
-Wno ignored-qualifiers is needed by core_log_function
rest deleted
<!-- Reviewable comment -OMQm9fBEm3lLDC4-Y2m:b-3ywo1b -->
<!-- Sent from Reviewable.io -->

---

ASSERT*() work but test doesn't build up
-Wno-unused-parameter is needed to mocked function
-Wno ignored-qualifiers is needed by core_log_function
rest deleted
<!-- Reviewable comment -OMQm9fBEm3lLDC4-Y2m:b-3ywo1b -->
<!-- Sent from Reviewable.io -->